packages/tools/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Tool, ToolType } from "./Tool";
+import { BoltTool } from "./BoltTool";
+import { LernaTool } from "./LernaTool";
+import { PnpmTool } from "./PnpmTool";
+import { RootTool } from "./RootTool";
+import { RushTool } from "./RushTool";
+import { YarnTool } from "./YarnTool";
+
+/**
+ * A convenient mapping of tool type names to tool implementations.
+ */
+const supportedTools: Record<ToolType, Tool> = {
+  bolt: BoltTool,
+  lerna: LernaTool,
+  pnpm: PnpmTool,
+  root: RootTool,
+  rush: RushTool,
+  yarn: YarnTool,
+};
+
+/**
+ * A default ordering for monorepo tool checks.
+ *
+ * This ordering is designed to check the most typical package.json-based
+ * monorepo implementations first, with tools based on custom file scchemas
+ * checked last.
+ */
+const defaultOrder: ToolType[] = [
```

Comment:
I think the default order is mainly related to finding the root and stuff like that so it makes sense to move it to `/find-root`. I wanted to know how that would feel like so I prepared a PR here: https://github.com/elliot-nelson/manypkg/pull/6

It's definitely not a hill that I want to die on but it would be kinda neat if all exports from `/tools` would actually be Tools. This way you can utilize `import *` better etc.

Reply:
Let me know what do you think

Reply:
Love it, merged into the PR!